bootstrap/peer.go

```diff
@@ -96,7 +96,7 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, config Config) (*P
 			return nil, errs.Combine(err, peer.Close())
 		}
 
-		peer.Transport = transport.NewClient(options)
+		peer.Transport = transport.NewClient(options, 20*time.Second)
```

Comment:
this look like a candidate for option flag or at least on const with default value. Something like `transport.NewClient(options, transport.DefaultRequestTimout)`.

Reply:
okay, I'm fine with adding that. can @egonelbre confirm that's cool before I change it?